hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This release fixes a problem introduced in `3.56.0 <v3.56.0>` where
+a low value for ``max_examples`` would result in a test failing with
```

Comment:
The markup is super verbose, but I do still like cross-references for settings to disambiguate them by both the styling and the link.